runtime/ftplugin/gitconfig.vim

```diff
@@ -13,3 +13,29 @@ setlocal formatoptions-=t formatoptions+=croql
 setlocal comments=:#,:; commentstring=;\ %s
 
 let b:undo_ftplugin = "setl fo< com< cms<"
+
+if !has('unix')
+  finish
+endif
+
+if !has('gui_running')
+  command! -buffer -nargs=1 Sman
+        \ silent exe '!' . 'LESS= MANPAGER="less --pattern=''^\s+' . KeywordLookup_gitconfig() . <q-args> . '\b'' --hilite-search" man ' . 'git-config' |
+        \ redraw!
+elseif has('terminal')
+  command! -buffer -nargs=1 Sman
+        \ silent exe 'term ' . 'env LESS= MANPAGER="less --pattern=''' . escape('^\s+' . KeywordLookup_gitconfig() . <q-args> . '\b', '\') . ''' --hilite-search" man ' . 'git-config'
+else
+  finish
+endif
+
+if !exists('*KeywordLookup_gitconfig')
+function KeywordLookup_gitconfig() abort
+  let matches = matchlist(getline(search('\v^\s*\[\s*.+\s*\]\s*$', 'nbWz')), '\v^\s*\[\s*(\k+).*\]\s*$')
```

Comment:
Here's my version of finding the section of the manual to look at: https://github.com/benknoble/Dotfiles/blob/854f9498cb90af3b84cb961a9e97cf0009970f31/links/vim/autoload/bk/gitconfig.vim#L3

It builds a Vim pattern rather than a Less RE (which is probably a POSIX RE?), but could be adapted. It handles cases like `[core]`, `[mergetool "fugitive"]`, and `excludesFiles = …` correctly.

Reply:
Nice, using Vim's Man file reader avoids depending on `less`. I would opt for your solution, but https://github.com/tpope/vim-git/pull/87 was already rejected on seemingly general grounds